linebot/models/send_messages.py

```diff
@@ -61,6 +62,16 @@ def __init__(self, text=None, quick_reply=None, **kwargs):
 
         self.type = 'text'
         self.text = text
+        if emojis:
+            new_emojis = []
+            for emoji in emojis:
+                emoji_object = self.get_or_new_from_json_dict(
+                    emoji, Emojis
+                )
+                if emoji_object:
+                    new_emojis.append(emoji_object)
+            self.emojis = new_emojis
+        self.emojis = emojis
```

Comment:
Is this line correct?

Reply:
I added `else` before this line,
thanks for review!